mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -13323,3 +13323,60 @@ resource "google_container_cluster" "with_enterprise_config" {
 }
 `, projectID, clusterName, networkName, subnetworkName)
 }
+
+func TestAccContainerCluster_disableControlPlaneIP(t *testing.T) {
+  t.Parallel()
+
+  clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+  networkName := acctest.BootstrapSharedTestNetwork(t, "gke-cluster")
+  subnetworkName := acctest.BootstrapSubnet(t, "gke-cluster", networkName)
+
+  acctest.VcrTest(t, resource.TestCase{
+    PreCheck:     func() { acctest.AccTestPreCheck(t) },
+    ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+    CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+    Steps: []resource.TestStep{
+      {
+        Config: testAccContainerCluster_basic(clusterName, networkName, subnetworkName),
```

Comment:
If I'm understanding correctly, `testAccContainerCluster_basic` will have the new field as enabled `true` by default, and the update will disable?

Reply:
Assuming yes according to the documentation